src/sagas.js

```diff
@@ -618,13 +622,15 @@ function adjustVVFile ({ data, options }) {
     logDebug('secure-attention was not found, inserting ', text)
     data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\n${text}`) // ending \n is already there
   }
-
+  if (usbFilter) {
+    data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\nusb-filter=${usbFilter}\n${text}`)
```

Comment:
Is it possible that line 623 adds `text` and then `text` is unnecessarily added once again by this `replace()`?